dremio_client/dremio_simple_client.py

```diff
@@ -171,9 +174,33 @@ def votes(self):
     def create_user(self, json):
         return create_user(self._token, self._base_url, json, ssl_verify=self._ssl_verify)
 
+    #def get_user(self, uid):
+     #   return get_user(self._token, self._base_url, uid, ssl_verify=self._ssl_verify)
+
+    def delete_user(self, uid):
+        return delete_user(self._token, self._base_url, uid, ssl_verify=self._ssl_verify)
+
     def update_user(self, uid, json):
         return update_user(self._token, self._base_url, uid, json, ssl_verify=self._ssl_verify)
 
+    def get_all_users(self, startIndex=None , count=None ,query=None ):
```

Comment:
thanks for including pydoc! Can you add the same for the rest of the methods you added?